content/3-lifecycle/1-on-mount/angular-renaissance/pagetitle.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from "@angular/core";
+
+@Component({
+  standalone: true,
+  selector: "app-pagetitle",
+  template: `<p>Page title: {{ pageTitle }}</p>`,
+})
+export class PagetitleComponent implements OnInit {
+  pageTitle = "";
```

Comment:
Any reason not to use signal here?

Reply:
Not at all, probably just a fail from my massive copy pasting